docs/userguide/networking/get-started-overlay.md

```diff
@@ -207,7 +210,7 @@ Once your network is created, you can start a container on any of the hosts and
 
 2. Start an Nginx server on `mhs-demo0`.
 
-		$ docker run -itd --name=web --net=my-net --env="constraint:node==mhs-demo0" nginx
+		$ docker run -d --name=web --net=my-net --env="constraint:node==mhs-demo0" nginx
```

Comment:
STET. This was provided to me by a networking and I tested it. It works as I tested it. If we want to change, we can talk about it but unless I test it end to end with this change, I'm not making it at this late date in the release.